themes/default/layouts/partials/header.html

```diff
@@ -133,7 +133,7 @@
                                         <li>
                                             <div class="list-title">
                                                 <a href="/product/internal-developer-platforms/">
-                                                    <i class="fas fa-box-open fa-fw"></i>
+                                                    <i class="ffas fa-users fa-fw"></i>
```

Comment:
I think this needs to be this: 

```suggestion
                                                    <i class="fas fa-users fa-fw"></i>
```